jobbergate-api/jobbergate_api/apps/job_submissions/schemas.py

```diff
@@ -44,6 +53,7 @@ class JobSubmissionCreateRequest(BaseModel):
     job_submission_description: Optional[str]
     job_script_id: int
     slurm_job_id: Optional[int]
```

Comment:
also, this is not needed in the moment of creation because the agent updates it later so this parameter should be passed only when updating the job submissions

Reply:
Good point!

Reply:
Addressed in latest commit.